pkg/webhook/webhook.go

```diff
@@ -411,3 +439,25 @@ func (e *Validator) shouldSkipOrganization(org string) bool {
 	}
 	return true
 }
+
+func (e *Validator) handleNonSTSFiles(ctx context.Context, client *github.Client, owner, repo, sha string, nonSTSFiles []string) (*github.CheckRun, error) {
+	log := clog.FromContext(ctx)
+	cr, _, err := client.Checks.CreateCheckRun(ctx, owner, repo, github.CreateCheckRunOptions{
+		Name:        "Trust Policy Validation",
+		HeadSHA:     sha,
+		ExternalID:  github.String(sha),
+		Status:      github.String("completed"),
+		Conclusion:  github.String("failure"),
+		StartedAt:   &github.Timestamp{Time: time.Now()},
+		CompletedAt: &github.Timestamp{Time: time.Now()},
+		Output: &github.CheckRunOutput{
+			Title:   github.String("Non-STS YAML file(s)."),
+			Summary: github.String("Found non-STS YAML files in `.github/chainguard` directory:\n\n" + strings.Join(nonSTSFiles, "\n")),
```

Comment:
I don't see where you check for `.yaml`?

This would flag `.github/chainguard/README.md` IIUC

Reply:
Good catch, fixed.